actions_test.go

```diff
@@ -0,0 +1,160 @@
+package opslevel_test
+
+import (
+	ol "github.com/opslevel/opslevel-go/v2023"
+	"github.com/rocktavious/autopilot/v2022"
+	"testing"
+)
+
+func TestWebhookActionHeadersMarshalling(t *testing.T) {
```

Comment:
I explicitly made this test to ensure we are generating the headers information consistently.

Long term I think this is going to cause the biggest problem for the terraform implementation.  Do your best to try and only use the map[string]string on the Terraform side both for the config and for the state.

I'm pretty sure we'll beable to make that work better then storing the headers in string format as thats going to be a nightmare for Terraform.